### PR TITLE
subscribe fake user to group

### PR DESCRIPTION
**Pull request by RRicardotj**
All commented code are useful but not needed at this time.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ function App () {
         <Route path='/' exact>
           <Landing />
         </Route>
-        <Route path='/checkout/:productId' exact>
+        <Route path='/checkout/:productId/:groupId' exact>
```
[the remark this answers is not in the document]
Comment (RRicardotj) [reply src/App.js]: Yes but requires a change in the backend that get product information by groupId.

File: src/hooks/useSubscribeToGroupMutation.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { useMutation } from '@apollo/react-hooks'
+
+import subscribeToGroupMutation from '../apollo/mutations/subscribeToGroup'
+
+function useCheckoutMutation () {
```
[the remark this answers is not in the document]
Comment (RRicardotj) [reply src/hooks/useSubscribeToGroupMutation.js]: I didn't get this point. I'm not separating the logic, this is the old checkout logic

[the remark this answers is not in the document]
Comment (RRicardotj) [reply src/hooks/useSubscribeToGroupMutation.js]: I've combined both mutation in just one... The backend it's not completed yet

File: src/scenes/Checkout/index.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import React, { useEffect, useState } from 'react'
-import useCheckoutMutation from '../../hooks/useCheckoutMutation'
-import useProductQuery from '../../hooks/useProductQuery'
+// import useCheckoutMutation from '../../hooks/useCheckoutMutation'
```
Comment (abelosorio) [inline src/scenes/Checkout/index.js]: Can we remove this code? I'd love to not leave "dead lines" in our code. Thanks!

File: src/scenes/Landing/index.js [hunk 4]
```diff
@@ -9,6 +9,8 @@ import GroupCard from '../../components/GroupCard'
 const Landing = () => {
   const { groups } = useGroupsQuery()
 
+  console.log(groups)
```
Comment (abelosorio) [inline src/scenes/Landing/index.js]: Can we remove this one too, please? I'd like the console to be clean

**Pull request merged**